scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -536,3 +540,68 @@ private[scalding] case class FlowStepStrategies[A]() extends Semigroup[FlowStepS
       }
     }
 }
+
+/*                                                                             
+ * Run a pre-existing MapReduce job with Scalding.
+ * Note that it is up to the user to properly configure the job.                                      
+ */
+class MapReduceJob(args: Args) extends Job(args) {
+  val conf: Configuration = mode match {
+    case h: HadoopMode => h.jobConf
+    case _ => new Configuration
```

Comment:
does this make sense? Are we really going to be able to run this in local mode? If not, perhaps we should sys.error here, or make `conf: Option[Configuration]`